scenario/state.py

```diff
@@ -444,6 +446,14 @@ def broken_event(self) -> "Event":
         )
 
 
+_DEFAULT_IP = "42.42.42.42"
```

Comment:
Is there a reason to use this Korean ISP IP address as the default? Could we use something in [192.0.2.0/24, 198.51.100.0/24, or 203.0.113.0/24](https://datatracker.ietf.org/doc/html/rfc5737)?

Reply:
yes! thank you I was looking for that spec but couldn't find it anymore. :)